lib/hammer_cli_katello/content_view_environment.rb

```diff
@@ -0,0 +1,29 @@
+module HammerCLIKatello
+  class ContentViewEnvironment < HammerCLIKatello::Command
+    resource :content_view_environments
+
+    class ListCommand < HammerCLIKatello::ListCommand
+      resource :content_view_environments, :index
```

Comment:
This can be removed; it inherits the resource from line 3. I tested it on my devel box and it still functions and pulls in everything correctly.